message_definitions/v1.0/ras_a.xml

```diff
@@ -0,0 +1,124 @@
+        <description>Unknown geometry.</description>
+      </entry>
+      <entry value="1" name="POI_GEOMETRY_TYPE_SQUARE">
+        <description>Square.</description>
+      </entry>
+      <entry value="2" name="POI_GEOMETRY_TYPE_RECTANGLE">
+        <description>Rectangle.</description>
+      </entry>
+      <entry value="3" name="POI_GEOMETRY_TYPE_CIRCLE">
+        <description>Circle.</description>
+      </entry>
+      <entry value="4" name="POI_GEOMETRY_TYPE_CUBE">
+        <description>Cube.</description>
+      </entry>
+      <entry value="5" name="POI_GEOMETRY_TYPE_CUBOID">
+        <description>Cuboid.</description>
+      </entry>
+      <entry value="6" name="POI_GEOMETRY_TYPE_SPHERE">
+        <description>Sphere.</description>
+      </entry>
+      <entry value="7" name="POI_GEOMETRY_TYPE_CYLINDER">
+        <description>Cylinder.</description>
+      </entry>
+    </enum>
+    <enum name="POI_CLASSIFICATION">
+      <description>
+        Point-of-Interest classification.
+        Provides default classes (0 to 2147483647 reserved), but implementation-specific classes can be considered instead on the field where this classification is set.
+      </description>
+      <entry value="0" name="POI_CLASSIFICATION_UNKNOWN">
+        <description>Unknown classification.</description>
+      </entry>
+      <entry value="1" name="POI_CLASSIFICATION_GEO_MARKER">
+        <description>Geolocation marker.</description>
+      </entry>
+      <entry value="2" name="POI_CLASSIFICATION_LANDING_PAD">
+        <description>Landing pad.</description>
+      </entry>
+      <entry value="3" name="POI_CLASSIFICATION_CLEAR_LANDING_SPOT">
+        <description>Detected suited landing spot.</description>
+      </entry>
+      <entry value="4" name="POI_CLASSIFICATION_BUILDING">
+        <description>Building.</description>
+      </entry>
+      <entry value="5" name="POI_CLASSIFICATION_PERSON">
+        <description>Person.</description>
+      </entry>
+      <entry value="6" name="POI_CLASSIFICATION_VEHICLE">
+        <description>Vehicle.</description>
+      </entry>
+      <entry value="7" name="POI_CLASSIFICATION_INGRESS_PORTAL">
+        <description>Ingress portal.</description>
+      </entry>
+    </enum>
+  </enums>
+  <messages>
+    <message id="238" name="POI_REPORT">
+      <wip/>
+      <!-- This message is work-in-progress and it can therefore change. It should NOT be used in stable production environments. -->
+      <description>Reports information for a detected Point-of-Interest (POI).
+        This can be either set by an operator (through MAV_CMD_SET_POI_FOCUS) or through onboard software or external processes.
+        The receiving system should track the POI being reported based on its UID + time of detection + time of update.
+        Detection time happens once for the same UID, while time of update changes when a specific metadata of that POI gets changed (e.g. position).
```

Comment:
UID?

Reply:
Ah, I see this. I'd move it up the top of the message fields.

Reply:
Makes sense,